readthedocs/templates/projects/environmentvariable_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Environment Variables" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-environment-variables %}class="active"{% endblock %}
+
+{% block project-environment-variables-active %}active{% endblock %}
+{% block project_edit_content_header %}
+  {% blocktrans trimmed with name=environmentvariable.name %}
+    Environment Variable: {{ name }}
+  {% endblocktrans %}
+{% endblock %}
+
+{% block project_edit_content %}
+  <form method="post" action="{% url 'projects_environmentvariables_delete' project_slug=project.slug environmentvariable_pk=environmentvariable.pk %}">
```

Comment:
What do we want to communicate on Environment Variable _detail_ page to the user?

Reply:
We probably need to explain that "These environment variables are available to all build steps." This is another place to keep UI prose copy short.